Without this patch, fastfetch's CPU reporting breaks on certain systems,
notably the Nintendo Wii and Nintendo Wii U.  See the below PR for more information.
This patch is here to work around the issue until either of these happen, and a
release is made containing the change:
A) My PR gets merged
B) Upstream adresses it in some other way

https://github.com/fastfetch-cli/fastfetch/pull/1640
--- a/src/detection/cpu/cpu_linux.c	2025-03-25 02:34:16.853438325 +0000
+++ b/src/detection/cpu/cpu_linux.c	2025-03-25 03:11:57.773754741 +0000
@@ -608,7 +608,7 @@
 
     #if __ANDROID__
     detectAndroid(cpu);
-    #else
+    #elif !__powerpc__ && !__powerpc
     detectSocName(cpu);
     #endif
 
